rest-api/authentication/models/auth-models.js

```diff
@@ -0,0 +1,35 @@
+const db = require('../../../database/dbConfig');
+const bcrypt = require('bcryptjs');
+const { generateToken } = require('../middlewares/generateToken');
+
+function register(user) {
+    return db('users')
+        .insert(user)
+        .returning('*')
```

Comment:
Hmm. Not sure I'm familiar with the syntax here. Or `delete` below...

Reply:
In PostgreSQL you need to specify what to return otherwise it will return junk (SQL information) and the delete keyword is useful to delete keys from objects before sending them over to the user, doesn't affect the data at all